tests/virt/node/conftest.py

```diff
@@ -64,7 +66,8 @@ def hotplugged_vm(
         client=unprivileged_client,
         data_source=golden_image_data_source_scope_class,
         cpu_max_sockets=EIGHT_CPU_SOCKETS,
-        memory_max_guest=TEN_GI_MEMORY,
+        # s390x doesn't support maxGuest as it doesn't support hotplug
+        memory_max_guest=TEN_GI_MEMORY if nodes_cpu_architecture != S390X else None,
```

Comment:
out of curiosity, what happens if the field is still set? does it ruins something in tests?